_schemaV2.graphql

```diff
@@ -5353,14 +5384,24 @@ type Partner implements Node {
   categories: [PartnerCategory]
   collectingInstitution: String
   counts: PartnerCounts
+
+  # A list of the partners unique city locations
+  cities(size: Int = 25): [String]
```

Comment:
Is `cities` actually used and is the city data not also provided by the `locationsConnection`?

Reply:
https://github.com/artsy/emission/blob/master/src/lib/Scenes/Artwork/Components/PartnerCard.tsx#L100 There are a couple cases in Emission like this one where we just need a list of the cities. In this view we wouldn't need to do a pagination query if we can just query for the `Partner.cities`, also we can remove a lot of logic and this util completely - https://github.com/artsy/emission/blob/master/src/lib/utils/filterLocations.tsx

Reply:
Hmm, ok, but in that case does this really need a `size` argument?